.github/workflows/push_pr.yml

```diff
@@ -82,10 +81,59 @@ jobs:
           GOPATH: ${{ github.workspace }}
         run: make integration-test
 
-  test-build:
-    name: Test binary compilation for all platforms:arch
+  test-build-nix:
+    name: Test binary compilation and packaging for linux
     runs-on: ubuntu-latest
+    env:
+      GPG_MAIL: 'infrastructure-eng@newrelic.com'
+      GPG_PASSPHRASE: ${{ secrets.OHAI_GPG_PASSPHRASE }}
+      GPG_PRIVATE_KEY_BASE64: ${{ secrets.OHAI_GPG_PRIVATE_KEY_BASE64 }} # base64 encoded
```

Comment:
non NR users will not have secrets available, have you check if the steps needs the secrets? otherwise it will always fail for external PR

Reply:
perhaps the signing process is decoupled and can be deactivated , something similar as the docker logging  I guess

Reply:
Fixed in [#015ae92](https://github.com/newrelic/nri-apache/pull/84/commits/015ae92237e36a5fb7924620f3d04ee907dbb0a3)